### ISSUE TITLE
Rate limit

### ISSUE THREAD

**Issue by sushant-here** [crunchy-labs/crunchy-cli#365]
**Describe the bug**
Unable to download Dragon Ball series due to rate limit. Similar issue with Dragon Ball Z.

Can download the 'example' series (DARLING in the FRANXX) from the Readme. Perhaps a edge case needs to be handled with those?

**To Reproduce**
Steps / command to reproduce the behavior:
Running `crunchy-cli 3.3.4`. I get the following error when i try downloading individual episodes or entire series of dragon ball:...
```
crunchy-cli download https://www.crunchyroll.com/watch/G4VUQZ7MX/the-new-threat
```
Output
```
:: ✔ Logging in
:: ✔ Parsed url 1
:: ✔ Loaded series information for url 1                                                                         :: Dragon Ball Z Season 1 (Dragon Ball Z (English Dub))
:: 	1. The New Threat » S01E01
:: An error occurred: You've probably hit a rate limit. Try again later, generally after 10-20 minutes the rate limit is over and you can continue to use the cli (https://cr-play-service.prd.crunchyrollsvc.com/v1/manifest/G4VUQZ7MX/...............)
```

**Expected behavior**
Episode (or series) to download

**Screenshots**
N/A

**Client (please complete the following information):**
 - OS: Mac OS Sonoma (14.3.1)
 - Version: 3.3.4

**Comment by bytedream** [comment]
I'm gathering information on this one. For anyone experiencing this issue with other series, can you tell me the series + your country (this issue is region specific)?

**Comment by Kartatz** [comment]
I'm having the same issue with this series:

```
https://www.crunchyroll.com/series/G1XHJV0ZV/i-shall-survive-using-potions
```

I tried downloading it using VPNs from many different locations (US, DE, ES, BR), but got the same result on all of them. 

```
$ crunchy-cli --etp-rt ... archive https://www.crunchyroll.com/series/G1XHJV0ZV/i-shall-survive-using-potions
:: ✔ Logged in
:: ✔ Parsed url 1
:: → Series I Shall Survive Using Potions! is not available with ar-ME, ca-ES, en-IN, es-LA, id-ID, ja-JP, ko-KR, ms-MY, pl-PL, pt-PT, ta-IN, te-IN, th-TH, tr-TR, vi-VN, zh-CN, zh-CN, zh-TW subtitles
:: ✔ Loaded series information for url 1
:: I Shall Survive Using Potions! Season 1 (I Shall Survive Using Potions!)
:: 	1. Perks of Reincarnation, Please! » S01E01
:: 	2. My Goal Is a Safe, Worry-Free Life in Another World! » S01E02
:: 	3. I'm Gonna Go Speak My Mind at the Castle! » S01E03
:: 	4. I'm Going into Angel Mode to Fight Back! » S01E04
:: 	5. I'll Use the Goddess's Potions to Do Business! » S01E05
:: 	6. I'll Use My Cheat to Destroy an Army! » S01E06
:: 	7. The Goddess Descends to End the War! » S01E07
:: 	8. I'm Seen as a Beautiful Girl and Abducted! » S01E08
:: 	9. "I'm Going to Relax in a Hot Spring for a While! » S01E09
:: 	10. Going Back to My Original Goal in a New Country! » S01E10
:: 	11. I'll Find a Husband with a Treasure Hunt! » S01E11
:: 	12. I Shall Survive Using Potions! » S01E12
:: An error occurred: You've probably hit a rate limit. Try again later, generally after 10-20 minutes the rate limit is over and you can continue to use the cli (https://cr-play-service.prd.crunchyrollsvc.com/v1/manifest/G4VUQ1983/evs3/4a2e16eb5bdc483bb5cb1bbb8f0a5e32/assets/p/7ef4203b1491204f851f79abcc9f96cb_,5315219.mp4,5315220.mp4,5315218.mp4,5315217.mp4,5315216.mp4,2832.txt,.urlset/manifest.mpd?playbackGuid=08-fff394a3-4c9f-42c1-b8ba-08e13ab6ea61&accountid=5f6b59a2-e41e-5adb-97fc-aacfe5f6eef4)
```

**Comment by bytedream** [comment]
Thanks for your response, apparently not region specific.
And Crunchyroll begins providing actual subtitles for audio languages other than japanese, didn't knew that

**Comment by gomander** [comment]
> I'm gathering information on this one. For anyone experiencing this issue with other series, can you tell me the series + your country (this issue is region specific)?

I'm experiencing this rate limit issue as well trying to download Frieren: Beyond Journey's End using v3.3.4 on Windows 11 from Sweden.

Full command:
```
crunchy-cli download -a en-US -s en-US --universal-output --ffmpeg-preset av1-lossless --include-chapters --etp-rt "6...2" https://www.crunchyroll.com/series/GG5H5XQX4/frieren-beyond-journeys-end
```
Results in output:
```
:: √ Logged in
:: √ Parsed url 1
:: √ Loaded series information for url 1
:: Frieren: Beyond Journey's End Season 1 (Frieren: Beyond Journey's End (English Dub))
::      1. The Journey's End » S01E01
::      2. It Didn't Have to Be Magic... » S01E02
...
::      27. An Era of Humans » S01E27
::      28. It Would Be Embarrassing When We Met Again » S01E28
:: An error occurred: You've probably hit a rate limit. Try again later, generally after 10-20 minutes the rate limit is over and you can continue to use the cli (https://cr-play-service.prd.crunchyrollsvc.com/v1/manifest/G7PU4MZ1G/evs3/1358b2409c364a0f988421280f9db270/assets/p/de7b6d025c4117fb3bdf5c91e3b5a20b_,5303605.mp4,5303606.mp4,5303604.mp4,5303603.mp4,5303602.mp4,2791.txt,.urlset/manifest.mpd?playbackGuid=0...6&accountid=3...d)
```

**Comment by bytedream** [comment]
Fixed in [v3.3.5](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.3.5). Crunchyroll began embedding subtitles in the file which only contained the video streams before, this was causing the false-positive rate limit detection

**Comment by 1nfiniteZest** [comment]
> Fixed in [v3.3.5](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.3.5). Crunchyroll began embedding subtitles in the file which only contained the video streams before, this was causing the false-positive rate limit detection

Thank you so much!
